syntax = "proto2";

import "google/protobuf/descriptor.proto";
package bgs.protocol;

option java_package = "bnet.protocol";
option java_outer_classname = "MethodOptionsProto";
option optimize_for = CODE_SIZE;

message BGSMethodOptions {
  optional uint32 id = 1;
}

extend .google.protobuf.MethodOptions {
  optional .bgs.protocol.BGSMethodOptions method_options = 90000;
}


